### PR TITLE
Hello Microverse Project Pull Request

### PR DESCRIPTION
**Pull request by dara-soji**
In creating the project I added the following files  index.html, index.css

### CODE REVIEW

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Correct workflow ✔️ 
- Perfect .gitignore file ✔️
- No Linter Errors ✔️


 ## Required Changes ♻️

- [ ] **_🛑 Your Readme file is missing some important sections from the original template , kindly add all the missing sections updated with your project information 🙏_**








## Optional suggestions

- [OPTIONAL] **_The commit message should be written in the imperative tense and capitalized_**

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @dara-soji, 

Great job making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been well addressed.



 ## Highlights
:heavy_check_mark: Good commit messages
:heavy_check_mark: Good use of GitHub flow
:heavy_check_mark: LInters are correctly setup and passing
:heavy_check_mark: Descriptive PR title and description

 ## Required Changes ♻️

- Just as the previous reviewer mentioned, your README.md file is incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. So please add all the missing parts of the readme and fill in all the necessary contents :+1: +1:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:

- [ ] Starting your commit messages with a capital letter to make them look more professional wink
**eg: Added the mobile menu feature to the webpage**

- [ ] Fixing the MIT license [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). The link is broken hence it doesn't lead to your license. The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this link to learn how to add one to your project :wink:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -16,63 +39,191 @@
...
-    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```
Comment (huseyinbiyik) [inline README.md:24]: Good job with the customization of the Readme. Let's make sure all sections on the Readme template are present in yours as the previous reviewers requested. Kindly add the "Feature Features" section below and don't forget to add at least one feature.

File: README.md [hunk 2]
```diff
@@ -16,63 +39,191 @@
...
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+<!-- GETTING STARTED -->
 
+## 💻 Getting Started <a name="getting-started"></a>
 
-<!-- Features -->
+> Describe how a new developer could make use of your project.
```
Comment (huseyinbiyik) [inline README.md]: [OPTIONAL] Please remove these instructions which are targeting you and add your own custom information.

File: README.md [hunk 3]
```diff
@@ -16,63 +39,191 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (huseyinbiyik) [inline README.md]: [OPTIONAL] Please remove these instructions which are targeting you and add your own custom information.

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @dara-soji,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @dara-soji 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**